src/duk_bi_global.c

```diff
@@ -448,15 +449,27 @@ DUK_INTERNAL duk_ret_t duk_bi_global_object_eval(duk_context *ctx) {
 		return 1;  /* return arg as-is */
 	}
 
+#if defined(DUK_USE_DEBUGGER_SUPPORT)
+	/*  XXX: level is used only by the debugger and should never be present
+	 *  for eval() called from Ecmascript code. is there a way to assert for
+	 *  this?
+	 */
```

Comment:
I think it'd be just fine to comment shortly here that level is present only for debugger-initiated calls. The actual eval() binding has nargs==1 but it's not easy to check here.